docs/src/index.md

```diff
@@ -1,11 +1,8 @@
----
-layout: turing
-title: "The Turing language for probabilistic machine learning"
----
+# Turing.jl
 
 **Turing** is a *universal* probabilistic programming language with a focus on intuitive modelling interface, composable probabilistic inference and computational scalability.
 
-# What you get...
+## Features
 
 Turing provides **Hamiltonian Monte Carlo** (HMC) and **particle MCMC** sampling algorithms for complex posterior distributions (e.g. those involving discrete variables and stochastic control flows). Current features include:
```

Comment:
This line is redundant imho (everything that it states is also stated in the bullets below)